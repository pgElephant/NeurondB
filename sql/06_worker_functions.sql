-- Test background worker management functions
SELECT proname, pronargs FROM pg_proc 
WHERE proname LIKE 'neuran%' 
ORDER BY proname;

-- Test manual worker execution (should not crash)
SELECT neuranq_run_once();
SELECT neuranmon_sample();

