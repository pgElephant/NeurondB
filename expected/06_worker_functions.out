-- Test background worker management functions
SELECT proname, pronargs FROM pg_proc 
WHERE proname LIKE 'neuran%' 
ORDER BY proname;
 proname | pronargs 
---------+----------
(0 rows)

-- Test manual worker execution (should not crash)
SELECT neuranq_run_once();
ERROR:  function neuranq_run_once() does not exist
LINE 1: SELECT neuranq_run_once();
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
SELECT neuranmon_sample();
ERROR:  function neuranmon_sample() does not exist
LINE 1: SELECT neuranmon_sample();
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
